src/Shared/Recruit.Vacancies.Client/FeatureToggle/FeatureNames.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Esfa.Recruit.Vacancies.Client.FeatureToggle
+{
+    public static class FeatureNames
+    {
+        public const string EnableLevy = "EnableLevy";
```

Comment:
👍 I like `AllowLevyPayingEmployersOnly `. Not sure why feature toggles should only apply to websites though.

Reply:
Is it because you prefer the FeatureName definitions to be shared by way of being in the Client package/namespace? I can see a potential conflict if we were to declare feature names in the client namespace for inspection within the client and have separate duplicate feature names in the websites that are similar as we could end up with 4 different FeatureNames files and not know where to consume/declare. I think this is a complication introduced as the project has grown. Going to need to think about this more. Leave this as it is if it suits for this ticket.

The FeatureToggle evaluator service being moved to the client lib sounds fair in that we may in the future move the toggle configuration to be read from db etc.